nixos/release-small.nix

```diff
@@ -97,7 +98,7 @@ in rec {
     constituents = lib.flatten [
       [
         "nixos.channel"
-        "nixpkgs.tarball"
+        #"nixpkgs.tarball"   # times out: https://hydra.nixos.org/build/213230807
```

Comment:
I think you want to remove this before we land it.

Reply:
I will drop the last two commits before merging.

https://github.com/NixOS/nixpkgs/pull/209870#issuecomment-1487625275